syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -34,6 +34,18 @@ def __share_secret(self, plain_text, workers):
         PRZS.setup(workers)
         return shares_map
 
+    @staticmethod
+    def one_shares(players, ring_size, shape=(1,)):
```

Comment:
please remove this, we need the interface to be as small as possible, you can just use torch.ones().share()

Reply:
Yep - tried to follow the model from ```AST```, but we might refactor that